src/ducks/notifications/BaseNotificationView.js

```diff
@@ -48,6 +48,14 @@ class BaseNotificationView extends NotificationView {
       'bank-layout': bankLayout
     }
   }
+
+  getExtraAttributes() {
+    return {
+      data: {
+        source: 'Cozy'
```

Comment:
On parle de faire une app amirale qui sera Cozy. Pourquoi ne pas mettre Cozy Banks en source ? 

J'imagine que tu fais ça dans le cadre de la surcharge M. pour savoir si c'est Cozy ou une surcharge, c'est ça ?

Reply:
Yep, I think I will put the slug of the app and put "Cozy" only on the M override.

Reply:
I've made the change : https://github.com/cozy/cozy-banks/pull/2002/commits/53f2ab3c9540be1f5732290d4f5a68864dce82c8, thanks !